### PR TITLE
shafiq added portfolio section

### PR DESCRIPTION
**Pull request by shafiqraza**

### CODE REVIEW

**Comment by shafiqraza** [comment]
this changes is done by shafiq

File: index.html [hunk 1]
```diff
@@ -9,7 +9,8 @@
 <body>
     <nav>
         <ul>
-            <li><a href="#home_by_masood">Home </a></li>   <!-- Created by masood -->
+            <li><a href="#home_by_masood">Home </a></li>
+            <li><a href="#port_by_shafiq">Portfolio</a></li>   <!-- Created by masood -->
```
Comment (masoodazhar) [inline index.html]: its good

Comment (masoodazhar) [reply index.html]: but one thing should change here

File: index.html [hunk 2]
```diff
@@ -20,6 +21,12 @@
         <h1>This is from masood heading 1</h1>
         <p>3,693,492 online stock photos, vectors, and illustrations are available royalty-free.See online stock video clips3,693,492 online stock photos, vectors, and illustrations are available royalty-free.See online stock video clips3,693,492 online stock photos, vectors, and illustrations are available royalty-free.See online stock video clips</p>
     </div>
+    <div class="" id="port_by_shafiq"style="width:100%;background:#f1f1f1;text-align:center;">
+        <div class="" style="width:300px;display:inline-block;border: 1px solid #000;height: 300px;  margin-top:50px;margin-bottom: 50px"></div>
+        <div class="" style="width:300px;display:inline-block;border: 1px solid #000;height: 300px; margin-top:50px;margin-bottom: 50px"></div>
+        <div class="" style="width:300px;display:inline-block;border: 1px solid #000;height: 300px; margin-top:50px;margin-bottom: 50px"></div>
+
+    </div>
```
Comment (masoodazhar) [inline index.html]: nice try. good

**Pull request merged**